server/src/main/java/org/eclipse/openvsx/web/ShallowEtagHeaderFilter.java

```diff
@@ -16,16 +16,12 @@ public class ShallowEtagHeaderFilter extends org.springframework.web.filter.Shal
 
     protected boolean shouldNotFilter(HttpServletRequest request) throws ServletException {
         // limit the filter to /api/{namespace}/{extension}, /api/{namespace}/details
-        // and /api/{namespace}/{extension}/{version} endpoints
+        // and /api/{namespace}/{extension}/{version}, and /api/-/search endpoints
```

Comment:
@yiningwang11 Why generate an ETag for the `/api/-/search` endpoint?

Reply:
Hi, thanks for the review!
I added this to generate an ETag for the response because I have disabled the cache for the search endpoint that returns the homepage. I thought with an ETag coming with the response header, even though the cache is disabled, no content would still be returned if the client has the up-to-date response.